dockerfile

```diff
@@ -1,12 +1,63 @@
-FROM python:3.8-alpine
+FROM python:3-slim
```

Comment:
Hmm I think these would work but would I would have to install python and some other stuff and I think it would actually be bigger. Right now my image is about 1GB, with Firefox installed.